components/director/internal/domain/runtime_context/entity_test.go

```diff
@@ -0,0 +1,51 @@
+package runtime_context_test
+
+import (
+	"testing"
+
+	"github.com/kyma-incubator/compass/components/director/internal/domain/runtime_context"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestEntity_EntityFromRuntimeModel(t *testing.T) {
+	// given
+	modelRuntime := model.RuntimeContext{
```

Comment:
Could you rename `modelRuntime` to `modelRuntimeContext`?

Same applies for `entityRuntime` to `entityRuntimeContext` below.